clientonly/index.js

```diff
@@ -18,6 +18,9 @@
 		["address", "port"].forEach((key) => {
 			config[key] = getCommandLineParameter(key, process.env[key.toUpperCase()]);
 		});
+
+		// determine if "--use-tls"-flag was provided
+		config["tls"] = process.argv.indexOf(`--use-tls`) > 0;
```

Comment:
there are still lint errors because of the single quotes, try `config["tls"] = process.argv.indexOf("--use-tls") > 0;`

Reply:
```suggestion
		config["tls"] = process.argv.indexOf("--use-tls") > 0;
```